### PR TITLE
Updated instruction on how to run the project.

### PR DESCRIPTION
**Pull request by mars22**
Hi, I was able to use `yarn link` to connect `sim/pkg` to frontend. I hope this will be useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,60 +24,45 @@ First you'll need to generate the WASM bundle for the `sim` crate using
...
-$ ls dist 
+$ ls dist
   0.bootstrap.js 1.bootstrap.js bootstrap.js ea9e01c84ad93861b0ad.module.wasm
   index.html
 ```
 
-If you want the code to be automatically recompiled whenever there are any 
-changes, you can use`watchexec`.
+If you want the code to be automatically recompiled whenever there are any
+changes, you can use`watchexec`. Open new terminal end execute.
```
Comment (Michael-F-Bryan) [inline README.md]: `end` -> `and`

**Review by Michael-F-Bryan — CHANGES_REQUESTED** [review]
Thanks a lot for this :+1:

I tried it in the past and for some reason `yarn serve` wasn't recompiling the frontend whenever `sim` got recompiled... I've got a feeling I forgot the `yarn install` step, so it didn't wire the links up properly.

**Comment by Michael-F-Bryan** [comment]
@mars22 other than a minor wording change, I'm happy with the PR. After it's merged I'll update the corresponding blog post.

**Comment by mars22** [comment]
@Michael-F-Bryan thx.

**Review by Michael-F-Bryan — APPROVED** [review]

**Pull request merged**